### ISSUE TITLE
Server icon

### ISSUE THREAD

**Issue by atiris** [google/material-design-icons#1338]
Hi, I reopen this request because the options offered in the [Server icon issue from 2016](https://github.com/google/material-design-icons/issues/523) do not cover the general visual representation of the term server. 

If you tell someone the word "server", I assume most will imagine a tall cabinet full of "flat computers." This term is actually "server rack" but I think it is generally visually closer to the idea of the word "server".
The "Storage" icon is close, but the elements are separate, which is not the case for a server. If I showed someone the "Storage" or "DNS" icon, for a layman it's not something that would evoke a server (remember the last movie you saw and showed how to work with the server or how to hack into the server - this is what an ordinary person imagines). And for professionals, the server icon is associated with Visio.
[Server icons used in Visio](http://www.visguy.com/wp-content/uploads/2009/09/servers-servers.png)

For me, visually the closest thing is something like this:

[Server icon](https://www.123rf.com/photo_112693516_server-icon-black-filled-vector-illustration-server-symbol-on-white-background-can-be-used-in-web-an.html)
[Google rack](https://www.google.com/search?q=server+rack&source=lnms&tbm=isch&sa=X)
[Server icon](https://www.dreamstime.com/stock-illustration-server-icon-vector-filled-flat-sign-solid-pictogram-isolated-white-symbol-logo-illustration-pixel-perfect-image88391589)

Thank you for your consideration

**Comment by tphinney** [comment]
Makes sense to me. I would have happily reopened the old issue, but since you have all the suggested pics here, we can just stick with this one.

Here are storage:
<img width="67" alt="Screen Shot 2022-06-29 at 9 53 42 AM" src="https://user-images.githubusercontent.com/6548266/176492882-af65a406-99c3-4c5c-8fc1-d06bacb9ab05.png">

... and dns:
<img width="71" alt="Screen Shot 2022-06-29 at 9 53 12 AM" src="https://user-images.githubusercontent.com/6548266/176492888-082dcd44-e901-49c0-8bb7-fff4e83e2057.png">

It seems like the main problems with these existing icons are:
- not enough vertical orientation
- not enough racks
- the racks/pieces are disconnected; should be in an overall frame

Does that seem correct?

**Comment by atiris** [comment]
Yes, I think somehow. I imagine the server as a cabinet with partitions, of course, I don't know without preview if our ideas are the same, but I'm not a graphic designer, fortunately :)
